File: formula1-tournament-api/Models/Season.cs
namespace formula1_tournament_api.Models
{
    public class Season
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public User User { get; set; }
        public Guid UserId { get; set; }
        public List<Team> Teams { get; set; }
        public List<Driver> Drivers { get; set; }
        public List<Race> Races { get; set; }
    }
}

File: car-racing-tournament-web/src/app/models/season.ts
import { Driver } from "./driver";
import { Race } from "./race";
import { Team } from "./team";
import { UserSeason } from "./user-season";

export class Season {
  constructor(
    public id: string,
    public name: string,
    public userSeasons: UserSeason[],
    public teams: Team[],
    public drivers: Driver[],
    public races: Race[]
  ) {}
}
